package.json

```diff
@@ -8,8 +8,9 @@
     "test": "test"
   },
   "scripts": {
+    "start": "mocha tests --recursive --reporter nyan --watch",
     "test": "mocha tests --recursive --reporter nyan",
-    "start": "mocha tests --recursive --reporter nyan --watch"
+    "test:ci": "mocha tests --recursive"
```

Comment:
One could also do some `yarn`-foo here to reduce the repetition:

```json
{
  "scripts": {
    "start": "yarn test -- --watch",
    "test": "yarn test:ci -- --reporter nyan",
    "test:ci": "mocha tests --recursive"
  }
}
```

but `yarn` then prints the little thing that looks like

```bash
yarn test:ci v0.19.1
$ mocha tests --recursive
```

for each of the invocations, which is a little messy.